.travis.yml

```diff
@@ -22,10 +22,17 @@ matrix:
       env: SYMFONY_VERSION='2.5.*'
     - php: 5.3.3
       env: DEPENDENCIES='low'
+    - php: 5.6
+      env: SYMFONY_VERSION=2.8.*@dev
+    - php: 5.6
+      env: SYMFONY_VERSION="3.0.x-dev as 2.8"
```

Comment:
i borrow that in https://github.com/sonata-project/SonataIntlBundle/blob/master/.travis.yml#L43